.github/workflows/close-org-prs.yml

```diff
@@ -0,0 +1,44 @@
+name: Close PRs from Orgs
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  close-org-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if PR is from an org and close if so
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            const isFork = pr.head.repo.full_name !== pr.base.repo.full_name;
+            const forkOwnerType = pr.head.repo.owner.type;
+            if (isFork && forkOwnerType === 'Organization') {
```

Comment:
Why `isFork` but not `isOwner`? Could consolidate the style with `const isOwner = pr.head.repo.owner.type === 'Organization'`.

Reply:
I was only referring to the code style, not to the check itself. The check itself makes sense.

Reply:
done